### ISSUE TITLE
Alt text (code) wrongly displayed on image lightbox

### ISSUE THREAD

**Issue by LukaHarambasic** [elk-zone/elk#1264]
### Pre-Checks

- [X] Check that there isn't [already an issue](https://github.com/elk-zone/elk/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/elk-zone/elk/discussions) or join our [Discord Chat Server](https://chat.elk.zone).
- [X] Providing a screenshot or video to reproduce the issue or show visually what was meant.
- [ ] I am willing to provide a PR.

### Describe the bug

Heyho,
The alt text is displayed in the top right corner of an image. It works for regular text.

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/22857823/212921674-ebe61828-7377-458e-bb33-0907df04c768.png">

But on [this toot](https://elk.zone/mastodon.social/@noleli/109704795463898389), the author pasted the code from the screenshot as a markdown code block. 

**Toot**
<img width="593" alt="screenshot toot with code" src="https://user-images.githubusercontent.com/22857823/212919557-3d078dc5-3101-4534-83b9-eb3fb6d38a50.png">

**Broken alt text**
<img width="1680" alt="screenshot of the same toot opened where in the top left corner only CSS is written" src="https://user-images.githubusercontent.com/22857823/212919761-c5083ce5-343a-44fb-8829-13dbc14d7623.png">

**Content is there**
<img width="1680" alt="highlighting the content in the lightbox to show that its there but not correctly displayed" src="https://user-images.githubusercontent.com/22857823/212920892-0e8b6499-f46b-45bb-9f73-7e3806b41838.png">

## Solution?

So I think displaying the code here doesn't add value, a screen reader will get it form the alt text, and others can read it on the screenshot, and I can't think of another scenario where you add code as alt text without a screenshot. So what about just displaying "Code from the picture" instead of the actual code but keeping the alt attribute as it is?

### Reproduction video or screenshot

Open this [this toot](https://elk.zone/mastodon.social/@noleli/109704795463898389) and check the alt text.

<img width="1680" alt="image" src="https://user-images.githubusercontent.com/22857823/212919761-c5083ce5-343a-44fb-8829-13dbc14d7623.png">

### Additional Context

Firefox, Mac (M1) - I don't think that it's relevant, but happy to provide more info.it's relevant, but I would be

**Comment by alexzhang1030** [comment]
<img width="726" alt="image" src="https://user-images.githubusercontent.com/49969959/212936271-f161c6c9-57fe-4e18-9ab5-affd86ccfc0f.png">

I think it's written by author.

**Comment by LukaHarambasic** [comment]
The colorfull part in your screenshot is an image, and the alt is a markdown code block

**Comment by alexzhang1030** [comment]
> The colorfull part in your screenshot is an image, and the alt is a markdown code block

I guess the author wrote it on purpose 🤣 

Cause Elk don't add extra things

**Comment by LukaHarambasic** [comment]
Yes, I never assumed that Elk added this. But Elk is responsible for displaying it - and the displaying is broken :)

**Comment by ZMYaro** [comment]
Alt text should not need to be visible by default if the image loads successfully.  I would be OK with the alt text being behind an [alt] button that displays a popover with as many lines as necessary, like in Alex's screenshot.
